hnn_core/network.py

```diff
@@ -219,47 +286,411 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
-    def _add_external_feeds(self):
-        """Legacy function, for backward compatibility with original HNN-GUI
+    def add_evoked_drive(self, name, mu, sigma, numspikes, weights_ampa,
+                         weights_nmda, location, space_constant=3.,
+                         dispersion_time=0.1, seedcore=42):
```

Comment:
should `sync_evinput` be another parameter here? Happy to delegate it to another PR though

Reply:
I was hoping we could document ourselves out of it: setting `sigma=0` has the desired effect.

Reply:
Try with the example and see if it works :)

Reply:
It does (see latest push), but it isn't pretty... Makes me think we have to have something like

```python
net = Network(params)
net.add_XXX_drive( ... )
...
clone_net = Network(clone=net)
clone_net.modify_XXX_drive(dynamics=new_dynamics)
...
```

Ugh, that'll be massive though :(

Reply:
maybe but for now it's fine as is. We can try to improve the narrative in the docs (perhaps copy some text from the main HNN tutorial?) and it will be fine. It's not a checkbox like in the GUI but it's also a matter of just changing one parameter. You don't have to redefine the weights down below, and then remove the "silent drives" I think it will already look much better.